### PR TITLE
WD : Feat grid : Added Grid Component

### PR DESCRIPTION
**Pull request by devananth**
## Added Grid Component

         - grid 2 column layout ( 50:50 )
         - grid 3 column layout 

**Live Link :** https://deploy-preview-13--wonderful-wescoff-6f3f26.netlify.app/components/grid/grid

### CODE REVIEW

**Review by ajay-mandaviya — APPROVED** [review]

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  <body>
+    <section class="grid-layout">
+      <header class="nav__header d-flex align-center space-bw gap-1">
+        <span
+          class="icon d-flex xy-center txt-bold rounded ml-1 hamburger-menu"
+        >
+          <i class="fas fa-bars"></i>
+        </span>
+        <div class="header__left ml-1 d-flex align-center gap-1">
+          <!--  Todo : Add Logo -->
```
Comment (Prabhav-Sharma) [inline Components/Grid/grid.html:24]: Are you yet to add logo?

**Review by Prabhav-Sharma — COMMENTED** [review]
I believe you're creating a component then adding that HTML link to each page you've already made which leads to a higher number of files changed in each PR. I would suggest that you create all the HTML files and add the basic structure with links to all the pages, which would help keep the links error-free and easy to manage.

File: Components/Grid/grid.html (hunk 1 above)
Comment (devananth) [reply Components/Grid/grid.html:24]: Yes, need to do!

**Comment by devananth** [comment]
> I believe you're creating a component then adding that HTML link to each page you've already made which leads to a higher number of files changed in each PR. I would suggest that you create all the HTML files and add the basic structure with links to all the pages, which would help keep the links error-free and easy to manage.

Yeah! Will definitely do that from next project.

**Pull request merged**